backend/Watchdog.Core/Watchdog.Core.BLL/Services/MemberService.cs

```diff
@@ -171,9 +171,10 @@ public async Task<InvitedMemberDto> AddInvitedMemberAsync(NewMemberDto memberDto
             return new InvitedMemberDto { Member = member, StatusCode = response.StatusCode };
         }
 
-        public async Task<bool> IsMemberOwnerAsync(int id)
+        public async Task<bool> IsMemberOwnerAsync(int organizationId, int id)
```

Comment:
Why have you used orgId and userid instead of just finding member by memberId. We have a nice method getMember() from AuthenticationService on frontend, which can be usefull here